docs/sections/footer.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+  <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+  <script src="/cocoandeve-styleguides/js/main.js"></script>
+</head>
+
+<body class="">
+  <div class="container mt-4">
+    <h1>Footer</h1>
+  </div>
+  <footer class="pt-md-6 pb-md-6 pt-2 pt-lg-4 pb-1 mobile-wrapper">
+    <div class="container">
+      <div class="footer-grid">
+        <div class="footer-grid__newsletter">
+          <h5 class="h2">Newsletter</h5>
+          <p>Receive exclusive offers, promotions and beauty tips via email.</p>
+          <form>
+              <div class="input-group mb-3">
+                  <input type="text" class="form-control" placeholder="Enter your email" aria-label="Enter your email">
```

Comment:
type must be EMAIL, this will allow the browser to validate the format as email text